implementations/code_df/code_changes_lines.py

```diff
@@ -0,0 +1,128 @@
+from datetime import datetime
+
+import conditions
+import utils
+from commit import Commit
+
+
+class CodeChangesLines(Commit):
+    """
+    Class for Code_Changes_Lines
+
+    :param items: A list of dictionaries, each element a line from the
+        JSON file with Perceval data
+
+    :param date_range: A tuple which represents the start and end date of
+        interest
+
+    :param is_code:  list of CodeCondition objects
+        It is used to determine what comprises source code.
+
+    :param conds: A list of Commit objects. It is used to
+        exclude or include different kinds of commits, like
+        empty or merge commits, or commits made on the master
+        branch.
+    """
+
+    def __init__(self, items, date_range=(None, None),
+                 is_code=[conditions.Naive()], conds=[]):
+        super().__init__(items, date_range, is_code, conds)
+
+        self.df = self._add_lines_modified_cols(self.df)
+
+    def compute(self):
+        """
+        Count the number of lines added or deleted in the data fetched
+        by Perceval.
+
+        It computes the sums of the 'additions' and 'deletions' columns
+        in the DataFrame.
+
+        :returns modified: A tuple representing the number of lines
+            modified. modified -> (additions, deletions)
+        """
+
+        df = self.df
+        additions = df['additions'].sum()
+        deletions = df['deletions'].sum()
+
+        modified = (additions, deletions)
```

Comment:
The definiton for "modified" is "added" `"deleted", see description of the metric. Or am I wrong?

Reply:
Thanks for the clarification! Will change the implementation.

Reply:
Thanks!